### PR TITLE
Feat/disablebnr 325520

### PR DESCRIPTION
**Pull request by meta-dipanshu-sharma**
# COMPLETES #[SPARK-325520](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-325520)

## This pull request addresses

- disableBNR API in meetings plugin
- sample app changes for disableBNR
- unit tests for disableBNR API

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

---

### CODE REVIEW

**Comment by arun3528** [comment]
Can you fix the package lock file ,



over all the package looks good to me

**Review by Neeraj-swarnkar — APPROVED** [review]
Looks Good.

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -5775,4 +5775,39 @@ export default class Meeting extends StatelessWebexPlugin {
...
+    let isSuccess = false;
+
+    try {
+      if (typeof this.mediaProperties === 'undefined' || typeof this.mediaProperties.audioTrack === 'undefined') {
+        throw new Error("Meeting doesn't have an audioTrack attached");
+      }
+      this.mediaProperties.audioTrack = WebRTCMedia.Effects.BNR.disableBNR(this.mediaProperties.audioTrack);
+      const audioStream = MediaUtil.createMediaStream([this.mediaProperties.audioTrack]);
+
+      audioStream.addTrack(this.mediaProperties.audioTrack);
```
Comment (mkesavan13) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:5796]: This is redundant. We already add track in the `MediaUtil.createMediaStream([this.mediaProperties.audioTrack]);` method. Kindly remove

**Review by mkesavan13 — CHANGES_REQUESTED** [review]
I hope you went to each commit and ran linter again. Just one small change. Other than that, looks good to me.

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 1 above)
Comment (meta-dipanshu-sharma) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js]: Done this change and samples app change as well.

**Review by mkesavan13 — APPROVED** [review]
Thanks for removing the `Start Listening to Local Audio` button. I wrote that just to test for ourselves. Now that we have the whole feature running, we do not need it. 

Thanks again. Looks good. Approving the PR.

**Pull request merged**